src/Resources/views/CRUD/base_history.html.twig

```diff
@@ -52,40 +51,8 @@ file that was distributed with this source code.
                 </div>
             </div>
         </div>
-        <div id="revision-detail" class="col-md-7 revision-detail">
+        <div id="revision-detail" class="col-md-7 revision-detail" {{ stimulus_target('sonata-revision', 'preview') }}>
 
         </div>
     </div>
-
-    <script>
-        jQuery(document).ready(function() {
-
-            jQuery('a.revision-link, a.revision-compare-link').bind('click', function(event) {
-                event.stopPropagation();
-                event.preventDefault();
-
-                action = jQuery(this).hasClass('revision-link')
```

Comment:
Are you sure this action is not used for anything?

Reply:
it's used for the check `if(action == 'show'){` which is considered as potential dead code

Reply:
Its added in #2266 but logic is no different from `show` action. Behavior is the same, click by link should update `#revision-detail` block.